### DISCUSSION TITLE
perform an initial filtering of low count cells

### DISCUSSION THREAD

**Discussion by sylestiel** [stuart-lab/signac#1505] [General]
Hi @timoast,
# perform an initial filtering of low count cells
md.500 <- md.500[md.500$passed_filters > 500, ]

$passed_filters is not present in the 10XMultiome output from Cell Ranger.

What alternative can I use.

Thanks!

**Comment by timoast** [comment]
The goal is to filter out cells with low counts, you can use anything in the metadata that quantifies the total counts per cell.
